websites/N/Notespace/presence.ts

```diff
@@ -0,0 +1,49 @@
+const presence = new Presence({
+	clientId: "1193651901221306378",
+});
+
+const enum Assets {
+	icon = "icon",
+}
+
+presence.on("UpdateData", async () => {
+	const currentPath = document.location.pathname,
+		presenceData: PresenceData = {
+			details: "Twoja wiedza jest tu",
+			largeImageKey: Assets.icon,
+			startTimestamp: new Date(),
+			buttons: [
+				{ label: "Odwiedź stronę", url: "https://notespace.edu.pl/" },
+				{
+					label: "Odwiedź bieżącą stronę",
+					url: `https://notespace.edu.pl${document.location.pathname}`,
+				},
```

Comment:
Rather than a generic button, it would be better to have specific button actions on specific pages that a user might want to visit.
Ex:
* View Course
* View Survey
* etc.

Reply:
Alright i will think about what to put there